example/javascriptlib/module/6-executable-config/build.mill

```diff
@@ -0,0 +1,40 @@
+package build
+
+import mill._
+import mill.javascriptlib._
+
+object foo extends TypeScriptModule {
+  def executionFlags =
+    Task {
+      Map(
+        "inspect" -> "",
+        "max-old-space-size" -> "4096"
+      )
+    }
```

Comment:
These should probably go in `4-compilation-execution-flags` rather than here

Reply:
4-compilation-execution-flags represent typescript compiler options
6-executable-config represent node execution flags

Reply:
as agreed In our conversations. do you still want me to move it?

Reply:
Yes let's move it